src/main/java/org/openhds/OpenHdsRestApplication.java

```diff
@@ -51,6 +51,9 @@ public CommandLineRunner initWithSampleData(MasterDataGenerator masterDataGenera
             if (!environment.containsProperty(SAMPLE_DATA_SIZE_PROPERTY)) {
                 // start normally
                 // TODO: create default user on first time startup?
+            	
+            	// Added by RTM 
```

Comment:
I think we can let Git keep track of who changed the code, and save comments for things like TODO and explanations of confusing code.